schemas/SuperProperties.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "object",
+  "description": "Super Properties are properties associated with events that are set once and then sent with every capture call, be it a $pageview, an autocaptured button click, or anything else.",
```

Comment:
I wonder if we should reword the `set once` bit to avoid confusion with [`$set_once`](https://posthog.com/docs/product-analytics/user-properties#what-is-the-difference-between-set-and-set_once) given these are separate from user properties?